src/components/Home.jsx

```diff
@@ -54,7 +56,14 @@ const Home = () => {
         <h2>Our Services</h2>
         <div className="services-list">
           {services.map((service, index) => {
-            return (
```

Comment:
whole card is looking like a link... It should be card and when you hover over the card... a button should appear saying Book Now. When you click on that it will take you to the booking page... 

The top navigation should be also taking to the same page

Use routing